lib/obanalyze/oban_jobs.ex

```diff
@@ -57,9 +57,17 @@ defmodule Obanalyze.ObanJobs do
   defp filter(query, term) do
     like = "%#{term}%"
 
-    from oj in query,
-      where: like(oj.worker, ^like),
-      or_where: like(oj.args, ^like)
+    case Oban.config() do
+      %{engine: Oban.Engines.Basic} ->
+        from oj in query,
+          where: ilike(oj.worker, ^like),
+          or_where: ilike(type(oj.args, :string), ^like)
+
+      %{engine: Oban.Engines.Lite} ->
```

Comment:
```suggestion
      _ ->
```

Let's have it as a catch-all. I'm not sure if the engine for testing needs to be included, but this will cover it.